compiler/fir/analysis-tests/testData/resolve/expresssions/operators/assignOperatorOverload.kt

```diff
@@ -1,75 +1,62 @@
-var result: String = "Fail"
+// !LANGUAGE: +AssignOperatorOverloadForJvm
```

Comment:
It's better to use the same test file for both FIR and FE 1.0 (all tests which are in `testData/diagnostics/tests` are run both by FIR and FE 1.0)

Reply:
Deleted test from `/fir/analysis-tests/testData`, now there are just tests in `testData/diagnostics/tests`. Thanks for that, tests were duplicated before.